mteb/evaluation/evaluators/Image/Any2AnyRetrievalEvaluator.py

```diff
@@ -292,19 +329,26 @@ def __init__(
         k_values: list[int] = [1, 3, 5, 10, 20, 100, 1000],
         score_function: str = "cos_sim",
         encode_kwargs: dict[str, Any] = {},
+        is_pipeline_task: bool = False,
```

Comment:
Some other ideas to implement this. Not sure which one would be the best or integrate more easily with MTEB code:
- Create a `retriever_class`, with default value of `Any2AnyDenseRetrievalExactSearch`. For pipeline eval, it would be passed `retriever_class = Any2AnyDenseRetrievalExactSearch`
- Moving this logic to a `RetrieverFactory` with the `is_pipeline` argument